SUMMARY.md

```diff
@@ -6,6 +6,20 @@
     * [Data Sources](data-sources.md)
     * [Software](software.md)
 * [User Manual](user-manual.md)
+    * [Data Products](data-products.md)
+        * [Infrared head imaging](flir.md)
+        * [Template](template.md)
+        * Photochemical reflectance index
+        * [Fluorescence intensity imaging](fluorescence-intensity-imaging.md)
+        * Short-wave infrared
+        * [Visible light imaging](visible-light-imaging.md)
+        * Visible-near-infrared imaging
+        * Atmospheric carbon dioxide
+        * Downwelling radiance
+        * Downwelling photosynthetically active radiation
+        * Environmental conditions
+        * Spectral reflectance
```

Comment:
Spectral reflectance is a generic term - the Hyperspectral and multispectral (SWIR, VNIR, NDVI, PRI) all measure spectral reflectance (please confirm @czender )